python/BioSimSpace/Parameters/Protocol/_amber.py

```diff
@@ -134,6 +199,19 @@ def __init__(self, water_model=None, leap_commands=None, property_map={}):
         # Call the base class constructor.
         super().__init__(forcefield="ff99", property_map=property_map)
 
+        # Validate the tolerance.
```

Comment:
Could be useful to abstract these checks to `@property` and `@property.setter`, but this is fine as is, just a general point.